src/service/url-replacements-impl.js

```diff
@@ -793,6 +792,31 @@ export class GlobalVariableSource extends VariableSource {
       return getter(geo);
     });
   }
+
+  /**
+   * Returns cached uach signal if available, calculate it if not.
+   * @param {string} variable
+   * @param {!Window} win
+   * @return {!Promise<string>}
+   */
+  getUach_(variable, win) {
+    if (this.cachedUach_[variable]) {
```

Comment:
I think checking `if (variable in this.cachedUach_)` would make more sense. Otherwise a `false` value in the dict would fail this check.